.github/workflows/issue-management-stale-action.yml

```diff
@@ -47,9 +47,9 @@ jobs:
           exempt-issue-labels: "needs author feedback"
           days-before-pr-stale: -1
           days-before-pr-close: -1
-          days-before-stale: 365
-          days-before-close: 0
-          close-issue-label: stale
+          days-before-issue-stale: 365
+          days-before-issue-close: 0
+          stale-issue-label: stale
```

Comment:
it's the third line that was the bug

b/c the stale action automatically removes any `close-issue-label` from open issues at the start of processing (before checking exemptions), because close labels are meant to be added only when an issue is being closed.